browserassets/tgui/tgui.bundle.js

```diff
@@ -1 +1 @@
dren:"Transform Station"})})]})})})}},46669:function(e,t,n){"use strict";t.__esModule=!0,t.TurretControl=void 0;var o=n(39812),r=n(85952),a=n(71494),c=n(74814),i=n(32942),l=n(67113),d=function(e){for(var t=[],n=0;n<e;n++)Math.random()>.3?t.push("Kill. "):t.push("KILL. ");return t.map((function(e,t){return(0,o.createComponentVNode)(2,c.Box,{inline:!0,preserveWhitespace:!0,fontSize:(0,i.randInt)(11,25)+"px",children:e},t)}))};t.TurretControl=function(e,t){var n=(0,a.useBackend)(t),i=n.act,u=n.data,s=u.enabled,m=u.lethal,p=u.emagged,h=u.area,C=u.locked,N=function(e){i("setLethal",{lethal:e})},f=function(e){i("setEnabled",{enabled:e})};return(0,o.createComponentVNode)(2,r.Window,{title:p?"FATAL ERROR":"Turret control ("+h+")",theme:p?"syndicate":"ntos",width:400,height:160,children:(0,o.createComponentVNode)(2,r.Window.Content,{align:"center",children:(0,o.createComponentVNode)(2,c.Box,{py:"6px",children:[!p&&!C&&(0,o.createComponentVNode)(2,c.Box,{fontSize:"16px",children:[(0,o.createComponentVNode)(2,c.Section,{width:"70%",children:(0,o.createComponentVNode)(2,c.Stack,{children:[(0,o.createComponentVNode)(2,c.Stack.Item,{width:"50%",children:(0,o.createComponentVNode)(2,c.Button,{icon:"exclamation-triangle",fluid:!0,selected:s,onClick:function(){return f(!0)},children:"Enabled"})}),(0,o.createComponentVNode)(2,c.Stack.Item,{width:"50%",children:(0,o.createComponentVNode)(2,c.Button,{icon:"power-off",fluid:!0,selected:!s,onClick:function(){return f(!1)},children:"Disabled"})})]})}),(0,o.createComponentVNode)(2,c.Section,{width:"70%",children:(0,o.createComponentVNode)(2,c.Stack,{children:[(0,o.createComponentVNode)(2,c.Stack.Item,{width:"50%",children:(0,o.createComponentVNode)(2,c.Button,{icon:"bolt",fluid:!0,selected:!m,onClick:function(){return N(!1)},children:"Stun"})}),(0,o.createComponentVNode)(2,c.Stack.Item,{width:"50%",children:(0,o.createComponentVNode)(2,c.Button,{icon:"skull-crossbones",fluid:!0,selected:m,onClick:function(){return N(!0)},children:"Lethal"})})]})})]}),!p&&!!C&&(0,o.createComponentVNode)(2,c.Section,{children:"Panel locked, swipe ID card to unlock."}),!!p&&(0,o.createComponentVNode)(2,c.Box,{py:"20px",children:[(0,o.createComponentVNode)(2,c.Box,{align:"center",fontFamily:"Courier New",children:(0,l.glitch)("ERROR: UNABLE TO READ AUTHORIZATION",12)}),(0,o.createComponentVNode)(2,c.Box,{align:"center",style:{"font-size":"20px"},children:d(7)})]})]})})})}},2861:function(e,t,n){"use strict";t.__esModule=!0,t.WeaponVendor=void 0;var o=n(39812),r=n(2497),a=n(71494),c=n(74814),i=n(85952),l=n(67113);t.WeaponVendor=function(e,t){var n=(0,a.useBackend)(t).data,r=(0,a.useLocalState)(t,"filter-available",!1),u=r[0],s=r[1];return(0,o.createComponentVNode)(2,i.Window,{width:550,height:700,children:(0,o.createComponentVNode)(2,i.Window.Content,{children:(0,o.createComponentVNode)(2,c.Stack,{className:"WeaponVendor",vertical:!0,fill:!0,children:[(0,o.createComponentVNode)(2,c.Stack.Item,{children:(0,o.createComponentVNode)(2,c.Section,{
```

Comment:
```suggestion
!function();
```
my bad on the resolve